experimental/attestation_common/src/certificate.rs

```diff
@@ -53,9 +51,7 @@ const DEFAULT_DNS_NAME: &str = "localhost";
 /// Indicates whether to add a custom TEE extension to a certificate.
 #[derive(PartialEq)]
 pub enum AddTeeExtension {
-    /// Enum value contains a PEM encoded TEE Provider's X.509 certificate that signs TEE firmware
-    /// key.
-    Yes(Vec<u8>),
```

Comment:
This is not necessary any more, since the TEE cert should be handled by the Verifier and Generator trait impls.